airflow/providers/databricks/operators/databricks_repos.py

```diff
@@ -90,7 +90,7 @@ def __init__(
             self.git_provider = self.__detect_repo_provider__(git_url)
             if self.git_provider is None:
                 raise AirflowException(
-                    "git_provider isn't specified and couldn't be guessed" f" for URL {git_url}"
+                    "git_provider isn't specified and couldn't be guessed for URL {git_url}"
```

Comment:
`f` at beginning is missing

Reply:
ups

Reply:
I'll fix it in the next PR, it's not critical

Reply:
Nice catch @alexott!